uwblueprint/planet-read
# Modify User model to store email

## Notion ticket link



[Modify User model to store email](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=ba22ac93d09040ff8219c60ac7caabdb&p=9d2011f9d77945ecb7bd6f6f7926056f)



## Implementation description

- add email column to user table



## Steps to test

1. start with an empty db (easiest way to do this is `docker-compose down --volumes` to discard the db volumes) 
2. `docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db upgrade"`
3. verify that the email col exists 
```
docker exec -it planet-read_db_1 /bin/bash
mysql -u root -proot
USE planet-read;
SELECT * FROM users;
```
4. `docker exec -it planet-read_py-backend_1 /bin/bash -c "python -m tools.insert_test_data"`
5. log in and poke around. verify things are working



## What should reviewers focus on?

- does it work!!?

## Checklist
